edu.cuny.hunter.log.updatesite/pom.xml

```diff
@@ -0,0 +1,39 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+		<groupId>edu.cuny.hunter.log</groupId>
+		<artifactId>parent</artifactId>
+		<version>1.4.0-SNAPSHOT</version>
+	</parent>
+  <artifactId>edu.cuny.hunter.log.updatesite</artifactId>
+  <version>1.4.0-SNAPSHOT</version>
+  <packaging>eclipse-repository</packaging>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.eclipse.tycho.extras</groupId>
+        <artifactId>tycho-p2-extras-plugin</artifactId>
+        <version>${tycho-version}</version>
+        <executions>
+            <execution>
+              <id>add-to-update-site</id>
+              <phase>package</phase>
+              <goals>
+                  <goal>mirror</goal>
+              </goals>
+              <configuration>
+                  <source>
+                    <repository>
+                        <url>${project.build.directory}/repository</url>
```

Comment:
This is going to be a problem considering [our current update site location](https://raw.githubusercontent.com/ponder-lab/Logging-Level-Evolution-Plugin/master/edu.cuny.hunter.log.updatesite).